### PR TITLE
#10, added README, demo and method code

### PR DESCRIPTION
**Pull request by shershen08**

### CODE REVIEW

File: src/InputTag.vue [hunk 1]
```diff
@@ -18,26 +27,36 @@
         type: Boolean,
         default: false,
       },
+      validate: {
+        type: String,
+        default: '',
```
Comment (matiastucci) [inline src/InputTag.vue]: maybe the default should be `'text'`. What do you think?

**Comment by shershen08** [comment]
Since validation is not required option - ignoring this attribute on the component should lead to always passing validation. If we place 'text' as a default - then it's only a-z allowed. So input like 'foo42' or just number or email or smth else won't pass.

**Comment by shershen08** [comment]
So I thought that it could be more handy to add more examples with other validation options so ppl could make use of this attribute. But  to start of simple with no validation at all.

**Comment by matiastucci** [comment]
yeah, I agree. Maybe in the future we can add a `pattern` prop too, where the user can pass his own regex. Good work!

**Pull request merged**